package/network/services/umdns/files/umdns.init

```diff
@@ -18,6 +18,11 @@ load_ifaces() {
 }
 
 reload_service() {
```

Comment:
is this function even needed ? that should be default behaviour if the function was not present

Reply:
Yes, we want umdns to be restarted on reloads. Otherwise, reload is useless.